### DISCUSSION TITLE
automated reporting of interactions or pairwise tests

### DISCUSSION THREAD

**Discussion by hcuve** [easystats/report#369] [General]
Thanks for such an amazing suite of tools.

In many cases when fitting regression models with categorical variables with more than two levels or when you have interactions,
it's desirable to probe the model results for pairwise comparisons or test interactions further, for example using means.
is there a way to get automated reporting for these? For example I have mixed models fitted in lmer where I first compute the anova on the model, and then run easystats::report on the anova results. However, if I probe pairwise or interactions (e.g. using emmeans, these don't work in easystats::report, is there a way around this.

many thanks

**Comment by rempsyc** [comment]
Continued in #370
